index.html

```diff
@@ -2132,6 +2132,11 @@ <h2>Definition of Roles</h2>
           For example, one common convention is that <kbd>Down Arrow</kbd> moves focus from the input to the first focusable descendant of the popup element.
           If the popup element supports <pref>aria-activedescendant</pref>, in lieu of moving focus, such keyboard mechanisms can control the value of <pref>aria-activedescendant</pref> on the <code>combobox</code> element.
           When a descendant of the popup element is active, authors MAY set <pref>aria-activedescendant</pref> on the <code>combobox</code> to a value that refers to the active element within the popup while focus remains on the <code>combobox</code> element.
+        </p>
+        <p>
+          User agents MUST compute a value for the <code>combobox</code>.
+          If the <code>combobox</code> element is also a host language element that provides a value, such as an HTML <code>input</code> element, user agents MUST set the value of the combobox to that value.
+          Otherwise, user agents must compute the value of the <code>combobox</code> from the descendant content of the <code>combobox</code> element.
```

Comment:
```suggestion
          Otherwise, user agents MUST compute the value of the <code>combobox</code> from the descendant content of the <code>combobox</code> element.
```

Reply:
Computing the value from the descendant content -- how? At first I assume it was copying a value from a descendant.

Reply:
> Computing the value from the descendant content -- how? 

And exposing it where? I don't see how user agents can know what to implement based on the current text.

Reply:
I'm about to make a commit that rewords this so this will become obsolete thread. The commit answers Aaron's question but not Joanie's.

Perhaps core-aam will someday address how to map the value of elements that have a value. But, James Craig and Aaron have explained to me that the lack of specification in AAM for things like the value of textbox (and combobox) is just an artefact of the fact that AAM is known not to be complete.  The AAMs have been built just as much by documenting what user agents do as opposed to specifying everything they need to do.

Perhaps in ARIA 1.3 we could add a valuedby attribute and address the core-aam gap. And, at the same time, we could write some language in the naming spec that specifies how valuedby computes. In the meantime, we don't need that to complete this requirement; we can rely on browser implementers to map value to accessibility APIs the same way they do now for textbox elements.